go.mod

```diff
@@ -19,13 +19,14 @@ require (
 	github.com/ojii/gettext.go v0.0.0-20170120061437-b6dae1d7af8a
 	github.com/op/go-logging v0.0.0-20160315200505-970db520ece7
 	github.com/pkg/errors v0.8.1-0.20180311214515-816c9085562c
+	github.com/prometheus/client_golang v1.1.0
 	github.com/snapcore/bolt v1.3.1 // indirect
 	github.com/snapcore/go-gettext v0.0.0-20191107141714-82bbea49e785 // indirect
 	github.com/snapcore/snapd v0.0.0-20200317200833-16631e228c07
 	github.com/yohcop/openid-go v0.0.0-20170901155220-cfc72ed89575
-	golang.org/x/crypto v0.0.0-20180409150046-beb2a9779c3b
-	golang.org/x/net v0.0.0-20180406214816-61147c48b25b
-	golang.org/x/sys v0.0.0-20180406135729-3b87a42e500a
+	golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2
```

Comment:
These libs are updated automatically because of introducing of `prometheus/client_golang`